tomviz/HistogramWidget.cxx

```diff
@@ -130,6 +132,8 @@ HistogramWidget::HistogramWidget(QWidget* parent)
 
   connect(&ActiveObjects::instance(), SIGNAL(viewChanged(vtkSMViewProxy*)),
           this, SLOT(updateUI()));
+  connect(&ModuleManager::instance(), SIGNAL(dataSourceRemoved(DataSource*)),
+	  this, SLOT(updateUI()));
```

Comment:
Off by one on indentation - should line up with line above (one more space than opening `(`)